### PR TITLE
Unify a set of common pseudo-instructions across codegens

### PR DESCRIPTION
**Pull request by fjeremic**
See each individual commit for which pseudo-instructions were unified. I did not touch x86 or ARM because they need to be taught to use `TR::InstOpCode` first and that is a much larger piece of work than what is done here. I will be tackling this once this PR is merged.

### CODE REVIEW

**Comment by fjeremic** [comment]
Jenkins build all

**Comment by fjeremic** [comment]
Jenkins build all

**Comment by fjeremic** [comment]
Jenkins build all

**Comment by fjeremic** [comment]
Jenkins build all

**Comment by fjeremic** [comment]
Jenkins build all

**Comment by fjeremic** [comment]
@dsouzai do you want to be a committer for this one?

**Review by dsouzai — APPROVED** [review]

File: compiler/aarch64/codegen/ARM64SystemLinkage.cpp [hunk 1]
```diff
@@ -627,7 +627,7 @@ TR::ARM64SystemLinkage::createEpilogue(TR::Instruction *cursor)
       }
 
    // return
-   cursor = generateRegBranchInstruction(codeGen, TR::InstOpCode::ret, lastNode, lr, cursor);
+   cursor = generateRegBranchInstruction(codeGen, TR::InstOpCode::retn, lastNode, lr, cursor);
```
Comment (knn-k) [inline compiler/aarch64/codegen/ARM64SystemLinkage.cpp:630]: This line must not be changed.
- `retn`: pseudo instruction
- `ret`: AArch64 instruction

Comment (fjeremic) [reply compiler/aarch64/codegen/ARM64SystemLinkage.cpp]: Right! This is a blatant bug from me. Fixed in [force-push](https://github.com/eclipse/omr/compare/247afa7987508aa12dc1695227443eff091179eb..b224aa3d87dc11b7aa4740c0b6b7523e7cd75d5b).

**Comment by fjeremic** [comment]
Jenkins build all

**Pull request merged**